keylime/httpclient_requests.py

```diff
@@ -33,12 +38,20 @@ def request(method, host, port, params=None, data=None, context=None,
             timeout=timeout)
 
     try:
+        if logger is not None:
```

Comment:
Is there a case where `logger` could equal `None`?

Reply:
Good catch. This was from a previous revision where I passed in the logger. Not needed now and removed.